dashboard/src/shared/types.ts

```diff
@@ -36,6 +36,13 @@ export class AppConflict extends Error {
   }
 }
 
+export class UnprocessableEntity extends Error {
+  constructor(message?: string) {
+    super(message);
+    Object.setPrototypeOf(this, new.target.prototype);
```

Comment:
I am not that proficient in JS. Why is ` Object.setPrototypeOf` this required, that's its purpose?

Reply:
So I stole this from https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-newtarget, honestly I don't fully understand it myself, but we _should_ understand it. @Angelmmiguel could you maybe look into if/why we need this?

Reply:
I see this is going to be tackled in #489